feature-libs/commerce-quotes/components/details/overview/commerce-quotes-details-overview.component.html

```diff
@@ -1,68 +1,80 @@
-<ng-container *ngIf="quoteDetails$ | async as quote">
-  <div class="cx-quote-overview">
-    <div class="container">
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(quote?.code, 'commerceQuotes.details.code') | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quote?.creationTime | cxDate,
-              'commerceQuotes.details.created'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quote?.updatedTime | cxDate,
-              'commerceQuotes.details.lastUpdated'
-            ) | async
-          "
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              quote?.previousEstimatedTotal.formattedValue,
-              'commerceQuotes.details.estimatedTotal'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          class="cx-card-description"
-          [content]="
-            getCardContent(
-              quote?.description,
-              'commerceQuotes.details.description'
-            ) | async
-          "
-          [truncateText]="true"
-          [charactersLimit]="30"
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              'commerceQuotes.states.' + quote?.state | cxTranslate,
-              'commerceQuotes.commons.status'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quote?.expirationTime | cxDate,
-              'commerceQuotes.details.expiryDate'
-            ) | async
-          "
-        ></cx-card>
+<ng-container *ngIf="quoteDetails$ | async as quoteDetailsState">
+  <ng-container *ngIf="!quoteDetailsState.loading; else loading">
```

Comment:
I think those should be replaced because in this case, the spinner will never show up as the data gets loaded and rendered so no point in showing spinner then

Reply:
Sorry, old name stayed and probably makes some misunderstanding - name of the stream should be `quoteDetailsState$`. I've changed return type to `Observable<QueryState<Quote | undefined>>` and as we're returning query state, first thing that I'm checking is if state is defined (what doesn't mean that data are loaded as well), then based on `loading` indicator I'm showing loading spinner or not. I'm pretty sure it reacts on each data reloading etc.